### PR TITLE
[TW-1142] Revamp reports

### PR DESCRIPTION
**Pull request by petewitcher-postman**
- Updated the reports docs based on redesigned reports. 
- Created new files and deleted old files. 
- Updated a link to the Token Scanner report, which will be deprecated soon.

### CODE REVIEW

**Review by jkonrath-postman — COMMENTED** [review]
(Not done yet, but publishing so I don't lose comments..)

File: src/pages/docs/administration/token-scanner.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ A scan starts whenever team members do any of the following actions:
 * Make changes to a collection or environment that's present in a public workspace.
 * Write new documentation for a Postman Collection and make it public.
 
-Postman delivers the scan results in the [Security audit reports](/docs/reports/security-audit-reports/) section of the [**Reports** dashboard](/docs/reports/reports-overview/).
+Postman delivers the scan results in the [Token scanner report](/docs/reports/security-audit-reports/) section of the [**Reports** dashboard](/docs/reports/reports-overview/).
```
[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/pages/docs/administration/token-scanner.md:49]: Confirmed with Shubham, `Token scanner` will not change to `Secret scanner` in reports.

File: src/components/LeftNav/LeftNavItems.jsx [hunk 2]
```diff
@@ -1207,35 +1207,35 @@ export const leftNavItems = [
...
-        name: 'Summary reports',
-        url: '/docs/reports/summary-reports/',
+        name: 'Overview reports',
+        url: '/docs/reports/overview-reports/',
       },
       {
-        name: 'Team details reports',
-        url: '/docs/reports/team-details-reports/',
+        name: 'Members overview reports',
+        url: '/docs/reports/members-overview-reports/',
```
[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/components/LeftNav/LeftNavItems.jsx:1219]: Looks good in Chrome and Firefox, no leftnav issues.

File: src/pages/docs/reports/all-apis-reports.md
Comment (jkonrath-postman) [inline src/pages/docs/reports/all-apis-reports.md:1]: Add a redirect from this page to the overview page.

File: src/pages/docs/reports/all-workspaces-reports.md
Comment (jkonrath-postman) [inline src/pages/docs/reports/all-workspaces-reports.md:1]: Add a redirect in redirects.json from this page to the overview page.

File: src/pages/docs/reports/api-gov-reports.md
Comment (jkonrath-postman) [inline src/pages/docs/reports/api-gov-reports.md:1]: Add a redirect from the old name to the new name

File: src/pages/docs/reports/billing-overview-reports.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Non-deleted integrations** - The team's integration limit and usage over time. Postman allows teams to integrate with various third-party services and platforms. The number of concurrent integrations a team can have is limited based on their plan type. Currently, Enterprise teams have a limit of 100 concurrent integrations. Once this limit is reached, teams should contact Postman Sales to discuss further modifications.
+
+**Postman API** - A historical view of the team's Postman API call limit and the usage on a monthly basis. The Postman API allows users to programmatically access data stored in their Postman account, using command-line interfaces (CLI) or automation flows. Teams have a limit on the number of calls they can make to the Postman API based on their plan. Currently, Enterprise teams have a limit of 1 million API calls per month.
+
+**Custom Domains** - The team's limit on custom domains and usage over time. Custom domains allow teams to host their published API documentation on their own domain. The number of custom domains a team can have is limited by their plan. Enterprise teams can have up to 10 custom domains before needing additional blocks.
```
Comment (jkonrath-postman) [inline src/pages/docs/reports/billing-overview-reports.md:36]:
```suggestion
**Custom Domains** - The team's limit on custom domains and usage over time. Custom domains allow teams to host their published API documentation on their own domain. The number of custom domains a team can have is limited by their plan. 
```

File: src/pages/docs/reports/summary-reports.md
Comment (jkonrath-postman) [inline src/pages/docs/reports/summary-reports.md:1]: Add redirects for this and the team-details-reports.md and redirect back to reports-overview.md

**Review by jkonrath-postman — APPROVED** [review]
Some fixes and some redirects to add, but otherwise LGTM!

File: src/pages/docs/reports/summary-reports.md
Comment (petewitcher-postman) [reply src/pages/docs/reports/summary-reports.md:1]: Done

File: src/pages/docs/reports/content-activity-reports.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    url: "https://blog.postman.com/introducing-new-postman-reports-for-enterprises/"
+---
+
+> [**Reports are available on all Postman Enterprise plans.**](https://www.postman.com/pricing)
+
+The **Content activity** dashboard shows the current landscape of elements created and updated by your team, organized into a **Workspaces overview** section and an **APIs overview** section.
+
+## Contents
+
+* [**Workspaces overview**](#workspaces-overview)
```
Comment (mark-anthony-postman) [inline src/pages/docs/reports/content-activity-reports.md:20]: Remove bold from Contents items

File: src/pages/docs/reports/content-activity-reports.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+* [**APIs overview**](#apis-overview)
+* [**More reports**](#more-reports)
+
+## Workspaces overview
+
+You can switch between team and public workspaces with the **Report Type** dropdown menu in the upper right.
```
Comment (mark-anthony-postman) [inline src/pages/docs/reports/content-activity-reports.md:26]: "dropdown menu" should be "dropdown list". Looks like there are 4 instances on this page

File: redirects.json [hunk 6]
```diff
@@ -2162,6 +2162,26 @@
...
+  {
+    "from": "/docs/reports/summary-reports/",
+    "to": "/docs/reports/reports-overview/"
+  },
+  {
+    "from": "/docs/reports/team-details-reports/",
+    "to": "/docs/reports/reports-overview/"
+  },
+  {
+    "from": "/docs/reports/all-apis-reports/",
```
Comment (mark-anthony-postman) [inline redirects.json:2175]: For some reason the last 3 redirects don't work for me when I test them, but they look fine, so I assume it's just my local dev server. Were you able to test them okay?

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/components/LeftNav/LeftNavItems.jsx [hunk 7]
```diff
@@ -1207,35 +1207,35 @@ export const leftNavItems = [
...
-        url: '/docs/reports/api-governance-reports/',
+        url: '/docs/reports/api-gov-reports/',
+      },
+      {
+        name: 'Billing overview reports',
+        url: '/docs/reports/billing-overview-reports/',
       },
       {
-        name: 'Token Scanner reports',
+        name: 'Token scanner report',
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/components/LeftNav/LeftNavItems.jsx:1238]: Also, is "Token Scanner" being changed to "Secret Scanner"? See Claire's PR: https://github.com/postmanlabs/postman-docs/pull/4865/files

[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/components/LeftNav/LeftNavItems.jsx:1238]: Confirmed with Shubham, reports is keeping the Token Scanner name until it is deprecated "in the coming months".

**Review by karishma-ag — APPROVED** [review]
LGTM

File: src/pages/docs/administration/token-scanner.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tapudp) [reply src/pages/docs/administration/token-scanner.md:49]: @petewitcher-postman also in near future we are deprecating it. Probably after September 15, 2023. Should there be any mention of that here?

File: src/pages/docs/collaborating-in-postman/public-api-network/sharing-your-workspace.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ Open the public workspace, then select the link icon <img alt="Link icon" src="h
 
 You can use the number of watchers and forks in your public workspace to track your API's success.
 
-> For more metrics, you can also view reports for [public workspaces](/docs/reports/all-workspaces-reports/#public-workspaces) and [public APIs](/docs/reports/all-apis-reports/#public-apis) in your [Postman reports dashboard](https://go.postman.co/reports/summary) ([Enterprise plans only](https://www.postman.com/pricing)).
+> For more metrics, you can also view reports for [public workspaces](/docs/reports/content-activity-reports/#public-workspaces) and [public APIs](/docs/reports/content-activity-reports/#public-apis) in your [Postman reports dashboard](https://go.postman.co/reports/summary) ([Enterprise plans only](https://www.postman.com/pricing)).
```
Comment (Tapudp) [inline src/pages/docs/collaborating-in-postman/public-api-network/sharing-your-workspace.md:35]: Shall we start using the new route now? 

```
[Postman reports dashboard](https://go.postman.co/reports/overview) 
```

**Review by Tapudp — COMMENTED** [review]
Left couple of comments. Overall it is good!

File: src/pages/docs/collaborating-in-postman/public-api-network/sharing-your-workspace.md (hunk 8 above)
Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/public-api-network/sharing-your-workspace.md:35]: Fixed

Comment (petewitcher-postman) [reply src/pages/docs/collaborating-in-postman/public-api-network/sharing-your-workspace.md:35]: Thanks Divyesh, I updated the link in sharing-your-workspace.md and also in reports-overview.md.

**Review by Tapudp — APPROVED** [review]
looks good!

**Pull request merged**